lib/models/blueprint.js

```diff
@@ -1231,76 +1232,19 @@ Blueprint.prototype.printBasicHelp = function(verbose) {
   } else {
     output += this.name;
 
-    var options = this.anonymousOptions;
+    output += printCommand.call(this, initialMargin, true);
```

Comment:
lets install the shared command on the respective prototypes, so they can be invoked off `this` rather then via call.